### ISSUE TITLE
Bad highlighting for ES6 code

### ISSUE THREAD

**Issue by bucaran** [github/linguist#2398]
If you look at the code in my repository at: 
https://github.com/bucaran/parsec/blob/master/src/lib/index.js

You will see it's not correctly highlighting the code at all.

Am I missing something or is this a bug?

**Comment by bucaran** [comment]
Ok, there is definitely a bug in the syntax highlight parser.

Basically, _with semicolon_:

```js
import stuff from "library";
class Color {}
```

Without it:
```js
import stuff from "library"
class BlackAndWhite {}
```

This affects me because I don't use semicolons in my JS code.

**Comment by pchaigno** [comment]
@bucaran Linguist only selects the grammar to use for highlighting. [atom/language-javascript](https://github.com/atom/language-javascript) is used to highlight JavaScript code on GitHub. You should open an issue there ;)

**Comment by bucaran** [comment]
Thanks.

**Comment by mnquintana** [comment]
@pchaigno This should've been fixed in the latest version of https://github.com/atom/language-javascript (0.77.0) - would you mind updating the submodule? :smile:

**Comment by pchaigno** [comment]
Oh great! I was just searching for when they fixed it :)
All submodules will be updated with the next version of Linguist (though not by me since I'm not from GitHub ;).

**Comment by mnquintana** [comment]
Oh haha sorry @pchaigno! The octocat avatar fooled me. :stuck_out_tongue_closed_eyes:

@arfon Would you prefer a PR to update the submodule?

**Comment by pchaigno** [comment]
I should have mentioned it earlier. Also, for repository admins, it says "Contributor" in the top right corner of their comments.

**Comment by pchaigno** [comment]
Looks fixed!

**Comment by bucaran** [comment]
Thanks. :+1:

**Comment by kentcdodds** [comment]
Can anyone think of a way to get this fixed in GitHub Enterprise? Even if it's possible for me to create a custom Chrome extension, I'd be good with that... Ideas appreciated :-)

**Comment by arfon** [comment]
@kentcdodds given that this was fixed on GitHub.com some time ago this should also be fixed on GitHub enterprise. Do you happen to know what version of GitHub Enterprise you are running?

**Comment by kentcdodds** [comment]
Ah, looks like we last updated about 6 months ago (when this was originally fixed) so we're on an old version (2.2.7 I think). Will ping someone about getting the update. People are threatening to change our code style back to using semicolons! This cannot happen! ;-)

Thanks!

**Comment by arfon** [comment]
OK, let me know if I can help in any way!

Cheers
Arfon
